build.gradle.kts

```diff
@@ -39,6 +39,7 @@ liquibase {
 dependencies {
     liquibaseRuntime(libs.liquibase.core)
     liquibaseRuntime(libs.mysql.connector.java)
+    liquibaseRuntime(libs.picocli)
```

Comment:
This is now required for liquibase gradle plugin (as per their readme), related issue #144